spec/support/coronavirus_landing_page_steps.rb

```diff
@@ -124,6 +141,16 @@ def then_i_can_see_the_timeline
     expect(page).to have_selector(".covid-timeline__item .gem-c-govspeak", text: "Face coverings are mandatory in shops")
   end
 
+  def then_i_can_see_the_timeline_for_england
```

Comment:
I think it'd be good to assert in here (and Wales) that it has the showing updates text too (e.g `expect(page).to have_text("Showing updates for England")`) since that seems quite a nuanced thing of JS / HTML / Ruby / CSS integrating.